kover-jvm-agent/src/main/java/kotlinx/kover/jvmagent/ParseUtils.java

```diff
@@ -0,0 +1,11 @@
+package kotlinx.kover.jvmagent;
+
+public class ParseUtils {
+    private ParseUtils() {
+        // no-op
+    }
+
+    public static boolean isBoolean(String value) {
```

Comment:
I think we can safely move this to `KoverJvmAgentPremain` without creating unnecessary utils class